### DISCUSSION TITLE
Fight'N Rage

### DISCUSSION THREAD

**Discussion by Cristobal15** [skyline-emu/skyline#756] [Game Requests]
Fight'N Rage is one of the best "Beat Em Up" style games within the Nintendo Switch console.

Currently in Skyline only the audio works in games with a black screen.

**Comment by Tskk-dev** [comment]
You can go to the discord for bug reports like this, everyone is welcome

**Reply by Cristobal15** [reply]
thanks 🤝, I already reported this.
